arches/app/models/migrations/11801_alter_spatialview_geometrynode.py

```diff
@@ -0,0 +1,174 @@
+# Generated by Django 5.2.1 on 2025-05-19 16:11
```

Comment:
You might rename the file to use the issue number, if that's still something the dev team cares about? (I do find the zigzag numbering confusing myself...)